Modules/Devices/test/Devices.API.Tests.Integration/StepDefinitions/GetIdentityListStepDefinitions.cs

```diff
@@ -0,0 +1,43 @@
+using Challenges.API.Tests.Integration.API;
+using Devices.API.Tests.Integration.Models;
+using Microsoft.Extensions.Options;
+    
+using static Challenges.API.Tests.Integration.Configuration.Settings;
+
+namespace Devices.API.Tests.Integration.StepDefinitions
+{
+    [Binding]
+    public class GetIdentityListStepDefinitions
+    {
+        private readonly RequestConfiguration _requestConfiguration;
+        private readonly IdentitiesApi _identitiesApi;
+        private HttpResponse<List<IdentityDTO>>? _identitiesResponse;
+
+        public GetIdentityListStepDefinitions(IOptions<HttpConfiguration> httpConfiguration, IdentitiesApi identitiesApi)
```

Comment:
No need to inject `httpConfiguration`. It's not used anyway, as far as I can see.